client/app/dbaas/dbaas-metrics/metrics.constant.js

```diff
@@ -11,8 +11,9 @@ angular.module("managerApp")
             }
         ],
         protos: [
-        // 'prometheus',
-        // 'influxdb',
+            "prometheus",
+            "influxdb",
+            "graphite",
             "warp10",
```

Comment:
doesn't this exist in some API enum?  Approving anyway, just check to be sure.

Reply:
I had put this in commentary a while ago. This should works.